InformationScripting/src/query_framework/QueryResultVisualizer.cpp

```diff
@@ -113,11 +119,24 @@ Optional<int> QueryResultVisualizer::visualize(const TupleSet& ts)
 			item->addOverlay(overlay, HIGHLIGHT_OVERLAY_GROUP);
 		}
 	}
+
+	auto htmlTuples = ts.tuples("html");
+	if (htmlTuples.size()) {
+		QString url = (*htmlTuples.begin())["html"];
+		// TODO an overlay or something would probably be better for this:
+		// TODO position it a bit better?
+		Comments::VCommentBrowser* browser = new Comments::VCommentBrowser{nullptr,
+				QUrl::fromLocalFile(QFileInfo{url}.absoluteFilePath())};
+		browser->setZValue(Visualization::Item::LAYER_OVERLAY_Z);
+		Visualization::VisualizationManager::instance().mainScene()->addTopLevelItem(browser);
```

Comment:
From the scene you can get the main view and from it you can see what's the currently displayed scene rect which you can use to position this new item right in the middle of the user's view.